src/fontra/views/editor/editor.js

```diff
@@ -384,6 +386,20 @@ export class EditorController {
       // {"key": "sourceIndex", "width": "2em"},
     ];
     this.sourcesList = new List("sources-list", columnDescriptions);
+    // TODO: relocate those to somewhere more appropriate after implementation
+    const addSourceCallback = () => {
+      console.log("add a source");
+    };
+    const removeSourceCallback = () => {
+      console.log("remove a source");
+    };
+
+    this.includeAddRemoveButtons(
```

Comment:
Suggest to inline the `includeAddRemoveButtons` method, and add its "listSelectionChanged" logic to the existing event listener.

Reply:
What if we want to use it for more lists though? That was my intent initially.

Reply:
It is not a lot of code and currently actually contains some code that is specific to the sources list.

Reply:
That method already contains logic that is specific to the sources list. We currently don't want to use it for other lists, so that's a bit of a premature generalization.